frontend/files/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "client",
```

Comment:
@h4ck3rk3y you can ignore all the files inside `frontend/files` because I copy and paste all of these from [here](https://github.com/BenElferink/MERN-template)